QEfficient/cloud/infer.py

```diff
@@ -103,61 +85,55 @@ def main(
         assert (
             generated_qpc_path == qpc_dir_path
         ), f"QPC files were generated at an unusual location, expected {qpc_dir_path}; got {generated_qpc_path}"
-        cloud_ai_100_exec_kv(
-            batch_size,
+    else:
+        #############################################
+        # hf model -> export -> compile -> execute
+        #############################################
+        # Load hf model
+        qeff_model = QEFFAutoModel.from_pretrained(pretrained_model_name_or_path=model_name, cache_dir=cache_dir, hf_token=hf_token)
```

Comment:
I believe for the qpc/onnx exists, it skips the `hf_download`, I mean if the user deletes cache-dir after 1st infer iteration, he will still be able to use qpc for execution on infer without need of cache-dir/checkpoints, just tokenizer should be enough.

Reply:
Yes, only tokenizer will be dowloaded in case cache_dir is deleted and qpc/onnx is found